products/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import ProductDetailView 
+urlpatterns = [
+    path('/detail/<int:product_id>' , ProductDetailView.as_view())
+]
```

Comment:
`detail`이라는 단어를 붙여줄 필요없이 아래와 같이 표현할 수 있습니다.
```python
:8000/products/1
:8000/products/2

```
```suggestion
from django.urls import path

from .views import ProductDetailView 

urlpatterns = [
    path('/<int:product_id>' , ProductDetailView.as_view())
]
```